dac/backend/src/main/java/com/dremio/dac/support/BasicSupportService.java

```diff
@@ -625,12 +747,33 @@ protected QueryProfile getProfile(SupportRequest supportRequest, int attempt) th
     return jobsService.get().getProfile(request);
   }
 
+  /**
+   * Retrieves a profile based on a support request and writes it as JSON to a file.
+   *
+   * @param out            a connection to the zip file where the JSON file will be written
+   * @param supportRequest the support request containing the User ID and Job ID
+   * @param attempt        the number of performed attempts to retrieve the support request profile
+   * @return               the retrieved support request profile
+   * @throws IOException          if an error occurs during serialization
+   * @throws JobNotFoundException if it doesn't find the job related to the support request
+   */
   private QueryProfile recordProfile(OutputStream out, SupportRequest supportRequest, int attempt) throws IOException, JobNotFoundException {
     QueryProfile profile = getProfile(supportRequest, attempt);
     ProtobufUtils.writeAsJSONTo(out, profile);
     return profile;
   }
 
+  /**
+   * Retrieves a profile based on a support request and writes it as JSON to a file.  It uses a Job ID generated based on the query ID.
```

Comment:
Use the `<p>` tag to separate paragraphs.

Reply:
done